build/README.md

```diff
@@ -14,188 +16,380 @@ To run a local DSS instance for testing, evaluation, or development, see [dev/st
 
-1.  Follow the steps below before running the final command if you are joining
-    existing clusters.
-
-1. To start the profiling service for the grpc-backend and/or http-gateway then enter the service
-   name for the profiler in your `main.jsonnet`. The name should be based on your cloud provider acceptable values. For Google cloud the name should meet the regex `^[a-z]([-a-z0-9_.]{0,253}[a-z0-9])?$`
-
 1.  Run `tk apply workspace/<CLUSTER_CONTEXT>` to apply it to the
     cluster.
 
-### Building Docker images
-
-Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
-
-Use the `build.sh` script in this directory to build and push an image tagged
-with the current date and git commit hash.
-
-## Joining an existing CockroachDB cluster
-
-Follow the steps above for creating a new CockroachDB cluster, but with the
-following differences:
-
-1.  In main.jsonnet, make sure you don't set shouldInit to true. This can
-    initialize the data directories on you cluster, and prevent you from joining
-    an existing cluster.
-
-1.  In main.jsonnet, add the host:ports of existing CockroachDB nodes to the
-    JoinExisting array. You should supply a minimum of 3 seed nodes to every
-    CockroachDB node. These 3 nodes should be the same for every node (ie: every
-    node points to node 0, 1, and 2). For external clusters you should point to
-    a minimum of 3, or you can use a loadbalanced hostname or IP address
-    of other clusters. You should do this for every cluster, including newly
-    joined clusters. See CockroachDB's note on the
-    [join flag](https://www.cockroachlabs.com/docs/stable/start-a-node.html#flags).
+1.  Wait for services to initialize.  Verify that basic services are functioning
+    by navigating to https://your-gateway-domain.com/healthy.
+    
+    - On Google Cloud, the highest-latency operation is provisioning of the
+      HTTPS certificate which generally takes 10-45 minutes.  To track this
+      progress:
+      - Go to the "Services & Ingress" left-side tab from the Kubernetes Engine
+        page.
+      - Click on the `https-ingress` item (filter by just the cluster of
+        interest if you have multiple clusters in your project).
+      - Under the "Ingress" section for Details, click on the link corresponding
+        with "Load balancer".
+      - Under Frontend for Details, the Certificate column for HTTPS protocol
+        will have an icon next to it which will change to a green checkmark when
+        provisioning is complete.
+      - Click on the certificate link to see provisioning progress.
+      - If everything indicates OK and you still receive a cipher mismatch error
+        message when attempting to visit /healthy, wait an additional 5 minutes
+        before attempting to troubleshoot further.
+
+1.  If joining an existing cluster, share your CRDB node addresses with the
+    operator of the existing cluster.  They will add these node addresses to
```

Comment:
@steeling or @charlie-pisuraj please verify this is what should happen -- basically, when a new DSS instance joins a DSS Region, the existing instances should be updated to point to the new instance's CRDB nodes, right?  Assuming so, is this the right way to do that?

Reply:
they'll also need to update the certificates provided to their cockroachdb backends to contain both certs

Reply:
Yep, as described on line 196 above?

Reply:
yup, missed that